### ISSUE TITLE
[ 1.10.2 ] TypeError: this._encodeParams is not a function

### ISSUE THREAD

**Issue by WayneUong** [meteor/meteor#11092]
After updating to meteor 1.10.2, we started to get this error everytime we make an `HTTP.get()` to the Mailchimp API/

```
error TypeError: this._encodeParams is not a function
error at _encodeParams (packages/url.js:69:23)
error at Object.exports.buildUrl (packages/url.js:91:16)
error at Function.exports._constructUrl (packages/url.js:119:17)
error at Object._call (packages/http.js:76:20)
error at Object.call (packages/meteor.js:306:21)
error at Object.HTTP.get (packages/http.js:217:20)
```

**Comment by WayneUong** [comment]
@filipenevola you can try this in a 1.10.2 meteor project:
```
HTTP.get('https://us17.api.mailchimp.com/3.0/campaigns', {
        headers: {
            authorization: 'Bearer 467032707d64785ca027be30484450a0'
        },
        params: {
            list_id: 'c9aeabaf17',
            type: [ 'regular', 'plaintext' ],
            count: 100,
            status: 'sent'
        }
    });
```

**Comment by WayneUong** [comment]
@filipenevola you can try this in a 1.10.2 meteor project:
```
HTTP.get('https://us17.api.mailchimp.com/3.0/campaigns', {
    headers: {
        authorization: 'Bearer 123'
    },
    params: {
        list_id: '123',
        type: [ 'regular', 'plaintext' ],
        count: 100,
        status: 'sent'
    }
});
```

**Comment by RobertLowe** [comment]
This is an issue

**Comment by RobertLowe** [comment]
Expanding on this, the issue seems larger, noticed this elsewhere:

```
Exception while invoking method 'someMethod' TypeError: this._addDefaultFieldSelector is not a function
    at Object.user (packages/accounts-base/accounts_common.js:119:53)
    at MethodInvocation.someMethod (imports/api/server/methods/someMethod.js:7:5)
    at MethodInvocation.methodMap.<computed> (packages/afrokick_kadira.js:2725:30)
    at maybeAuditArgumentChecks (packages/ddp-server/livedata_server.js:1771:12)
    at packages/ddp-server/livedata_server.js:719:19
    at Meteor.EnvironmentVariable.EVp.withValue (packages/meteor.js:1234:12)
    at packages/ddp-server/livedata_server.js:717:46
    at Meteor.EnvironmentVariable.EVp.withValue (packages/meteor.js:1234:12)
    at packages/ddp-server/livedata_server.js:715:46
    at new Promise (<anonymous>)
    at Session.method (packages/ddp-server/livedata_server.js:689:23)
    at packages/afrokick_kadira.js:2616:38
    at Meteor.EnvironmentVariable.EVp.withValue (packages/meteor.js:1234:12)
    at Session.sessionProto.protocol_handlers.method (packages/afrokick_kadira.js:2615:44)
    at packages/ddp-server/livedata_server.js:559:43
```

Really very odd https://github.com/meteor/meteor/blob/devel/packages/accounts-base/accounts_common.js#L119

I'll try to make a reproduction of this one as well, but something is weird with `this`.

**Comment by WayneUong** [comment]
@filipenevola  can you release this as a hotfix?

**Comment by filipenevola** [comment]
Hi @WayneUong, fixed on url@1.3.1.
